proto/spacemesh/mesh.proto

```diff
@@ -24,16 +26,21 @@ service MeshService {
     // Number of transactions per second (a network parameter)
     rpc MaxTransactionsPerSecond(google.protobuf.Empty) returns (SimpleInt);
 
-    ////////// Streams
+    // Get account data query
+    rpc AccountData(AccountMeshDataQueryParams) returns (AccountMeshDataResult);
```

Comment:
```suggestion
    rpc AccountData(AccountMeshDataQueryParams) returns (AccountMeshDataQueryResult);
```

Reply:
I think it is better and cleaner to use a bit field for filters and not a list of filters. It shouldn't be a problem to create them in any modern client / language. Yes, global state changes should be in its own prs to keep prs small and about less things...